apps/web/src/app/admin-console/organizations/core/views/group.view.ts

```diff
@@ -8,6 +8,10 @@ export class GroupView implements View {
   id: string;
   organizationId: string;
   name: string;
+  /**
+   * @deprecated
+   * To be removed alongside `FeatureFlag.FlexibleCollections`.
```

Comment:
This might be useful to clarify:

```suggestion
   * To be removed alongside `FeatureFlag.FlexibleCollections`.
   * This will always return `false` if Flexible Collections is enabled.
```